### DISCUSSION TITLE
Unable to make neurodesk work

### DISCUSSION THREAD

**Discussion by lucsanara** [neurodesk/neurodesk.github.io-hugo#447] [Q&A] [answered]
Hi!!

We have been working with neurodesk for long, though now it is arising an error we do not know how to solve. Each time we try to run any command (does not mind the pipeline), it shows the following (showing example with aslprep):  FATAL: exec /opt/conda/envs/aslprep/bin/aslprep failed: invalid argument
Can anyone shed light into this?

Thanks in advance!

**Comment by stebo85** [comment]
Interesting- can you give more details on how we could reproduce this problem? Does it also happen on our play instances?

**Comment by lucsanara** [comment]
We are running commands through neurodesk app in an intel Mac, and docker desktop version has been uploaded to 4.34.2. We can load the modules but the error arise as we try to do anything with them, for example aslprep -h is also showing the error.

**Comment by stebo85** [comment]
Ok, that could be a recent docker bug on Mac. Can you try podman and check if you get the same error?

**Comment by lucsanara** [comment]
Hi! We will give a try, though we have worked with other Macs with docker desktop now, and we do not have this problem...

**Reply by stebo85** [reply]
Do the other Macs have the same docker version and macOS version?

**Reply by lucsanara** [reply]
Hi! We finally solved it by downgrading docker version. Thanks for your answers and suggestions

**Reply by stebo85** [reply, marked as answer]
Yes, that's a good solution until docker fixes the bug with the next update :)

**Comment by stebo85** [comment]
Dear @lucsanara - we released a new version of the neurodesk app 1.6.1 which now has a workaround for this problem included, so you can revert to use docker.
